File: src/functions/exp1.py
import math

def main():
    print(exp(1))

def exp(x):
    n = 2147483647 # max value of integer, we'll treat this as infinity.
    sum = math.pow((1 + x/n), n)
    return sum

if __name__ == "__main__":
    main()

File: src/functions/exp1.go
package main

import (
    "fmt"
    "math"
)

func main() {
    fmt.Println(exp(1))
}

func exp(x float64) float64 {
    n := 2147483647.0 // highest value of float64
    sum := math.Pow((1 + x/n), n)
    return sum
}
